app/templates/ui/app/login/login.service.js

```diff
@@ -9,9 +9,10 @@
   function LoginService($http, $modal, $q, $rootScope, $state, $stateParams, messageBoardService) {
     var _loginMode = 'full'; // 'modal', 'top-right', or 'full'
     var _loginError;
-    var _isAuthenticated = false;
     var _toStateName;
     var _toStateParams;
+    var _isAuthenticated = isLoggedIn();
```

Comment:
It's currently returning the promise I think. At least the value isn't there in time for the bottom. Ideally I don't want to wait for a promise.